### DISCUSSION TITLE
X.509 Certificate

### DISCUSSION THREAD

**Discussion by cryptojane94** [CycloneDX/bom-examples#46] [Q&A]
Hello everyone!

I had a look at the example on the CBOM, but I am not sure on how to obtain this result for my repositories. Which tool you would suggest to have the CBOM of the keys and certificates I have on a folder?

**Comment by stevespringett** [comment]
I would recommend visiting the CycloneDX Tool Center and performing a search on `crypto` or `CBOM`. There are a few tools with many more in development.
